docs/source/guides/staking_guide.rst

```diff
@@ -47,6 +47,17 @@ All staking-related operations are performed through the ``nucypher stake`` comm
 +-----------------+--------------------------------------------+
 
 
+Staking Overview
+-----------------
+
+Most stakers will use the following commands in order:
+
+1. Create a new persistent stakeholder configuration
+2. Initialize a new stake
+3. List all Stakes
+4. Bond a worker to the stake
+5. Divide an existing Stake
```

Comment:
Is divide something that stakers will have to do? I still don't know what are good use cases for dividing stake. @michwill @szotov ?

Reply:
Maybe you're close to the max_stake and want to restake your rewards?

Reply:
when you want to extend/prolong only part of your stake